docs/index.mdx

```diff
@@ -5,9 +5,128 @@ sidebar_position: 0
 import QuickNavigation from "@site/src/components/QuickNavigation";
 import { items } from "@site/src/content/cardItems";
 
-# Welcome to Fishjam!
+# Welcome to Fishjam
 
-Here you will find everything you need to start building your multimedia streaming applications using our service.  
-Start from the beginning, get right away to setup or get to know our glossary.
+<p style={{ fontSize: "1.1rem" }}>
+Welcome to the Fishjam documentation!
+
+Fishjam is a multimedia streaming toolkit that allows you to build real-time
+video and audio streaming applications using WebRTC. We provide the
+infrastructure, media server, and client SDKs so you can focus on building
+your apps.
```

Comment:
I would love 😍 to see here a tagline, that would indicate to the users what they get from this particular site, which is Fishjam docs, not the landing page. 
```suggestion
Learn how to create live video and audio streaming applications with Fishjam, the all-in-one WebRTC toolkit. 
```